tests/indent/common.lua

```diff
@@ -28,6 +32,9 @@ local function same_indent(state, arguments)
 end
 
 local function format_indent(arg, fmtargs)
+  if not arg or not fmtargs then
+    return
+  end
```

Comment:
necessary if you do assertions inside the indent assertions (or have other kind of failures while checking indents)

Reply:
can be probably removed